core/src/main/java/org/infinispan/statetransfer/StateConsumerImpl.java

```diff
@@ -652,7 +651,7 @@ public void start() {
       isFetchEnabled = (mode.isDistributed() || mode.isReplicated()) &&
               (configuration.clustering().stateTransfer().fetchInMemoryState() || configuration.persistence().fetchPersistentState());
       //rpc options does not changes in runtime. we can use always the same instance.
-      rpcOptions = rpcManager.getRpcOptionsBuilder(ResponseMode.SYNCHRONOUS)
+      rpcOptions = rpcManager.getRpcOptionsBuilder(ResponseMode.SYNCHRONOUS_IGNORE_LEAVERS)
```

Comment:
Maybe you could rename that field to `syncIgnoreLeavers` to make other potential uses aware.